media.css

```diff
@@ -0,0 +1,1356 @@
+  }
+
+  .header__menu-bar {
+    display: flex;
+    justify-content: flex-end;
+    align-items: center;
+  }
+
+  .menu-bar {
+    display: block;
+    margin-right: 44px;
+  }
+
+  .link {
+    font-size: 20px;
+    line-height: 19px;
+    color: #fff;
+  }
+
+  .top-content {
+    background-image: url(img/header-img-768.png);
+    background-repeat: no-repeat;
+    background-size: cover;
+    width: 100%;
+    height: 415px;
+    margin-bottom: 67px;
+  }
+
+  .top-content__block {
+    padding-left: 43px;
+    padding-top: 120px;
+  }
+
+  .top-content__h1 {
+    font-size: 50px;
+    line-height: 48px;
+    margin-bottom: 37px;
+  }
+
+  .hidden-768 {
+    display: none;
+  }
+
+  .catalog {
+    width: 100%;
+  }
+
+  .catalog__title {
+    font-size: 30px;
+    line-height: 29px;
+  }
+  .catalog__items {
+    width: 626px;
+    margin: 0 auto;
+  }
+
+  .catalog__item {
+    width: 281px;
+    height: 352px;
+  }
+
+  .catalog__item:not(:last-child)::after {
+    content: "";
+    display: none;
+  }
+
+  .catalog-item__image {
+    width: 233px;
+    height: 230px;
+    margin-bottom: 24px;
+    padding-left: 0;
+  }
+
+  .catalog-item__title {
+    font-size: 20px;
+    line-height: 19px;
+    margin-bottom: 20px;
+    padding-left: 15px;
+  }
+
+  .catalog-item__description {
+    grid-template-columns: 1.5fr 1fr;
+    padding-left: 15px;
+    padding-right: 7px;
+  }
+
+  .catalog-item__price {
+    font-size: 25px;
+    line-height: 24px;
+  }
+
+  .catalog-item__grey,
+  .catalog-item__black,
+  .catalog-item__red,
+  .catalog-item__orange,
+  .catalog-item__brown,
+  .catalog-item__cherry,
+  .catalog-item__blue {
+    height: 20px;
+    width: 20px;
+  }
+
+  .video-section {
+    height: 756px;
+    width: 100%;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    background: linear-gradient(2.6deg, #000 92.7%, #fff 93%);
+    margin-bottom: 0;
+  }
+
+  .video-section__image {
+    display: flex;
+    justify-content: center;
+    order: 1;
+  }
+
+  .video {
+    display: none;
+  }
+
+  .video-adaptive {
+    display: block;
+    width: 90%;
+    height: auto;
+    margin: 0 auto;
+  }
+
+  .video-section__content {
+    display: flex;
+    flex-direction: column;
+    order: -1;
+    width: 88%;
+    align-items: center;
+  }
+
+  .video-content__h3 {
+    width: 100%;
+    font-size: 30px;
+    line-height: 37px;
+    margin-bottom: 15px;
+    text-align: center;
+  }
+
+  .video-content__description {
+    width: 100%;
+    font-size: 20px;
+    line-height: 25px;
+    margin-bottom: 35px;
+    text-align: center;
+  }
+
+  .stories-content {
+    width: 100%;
+    background-color: #000;
+    color: #fff;
+    margin-top: -5px;
+    margin-bottom: 0;
+    padding-bottom: 138px;
+  }
+
+  .stories-content__h4 {
+    font-size: 30px;
+    line-height: 29px;
+    text-align: center;
+    margin-bottom: 45px;
+    color: #fff;
+  }
+
+  .stories-items {
+    width: 88%;
+    
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/PolinaYartseva_axion/pullRequest?prid=5569682)